core-plugin/src/com/google/gct/idea/debugger/CloudDebugProcess.java

```diff
@@ -267,6 +270,9 @@ public void run() {
                 if (id.equals(myNavigatedSnapshotId)) {
                   if (result.getIsFinalState() != Boolean.TRUE || result.getStackFrames() == null) {
                     getBreakpointHandler().navigateTo(result);
+                    if (result.getStackFrames() == null) {
+                      navigateToBreakpoint(result);
```

Comment:
what's the difference between getBreakpointHandler().navigateTo() and navigateToBreakpoint()?